queue_processors/queue_processor/handle_message.py

```diff
@@ -0,0 +1,388 @@
+# ############################################################################
+# Autoreduction Repository :
+# https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################
+"""
+This modules handles an incoming message from the queue processor
+and takes appropriate action(s) based on the message contents.
+
+For example, this may include shuffling the message to another queue,
+update relevant DB fields or logging out the status.
+"""
+import datetime
+import logging.config
+import traceback
+
+from model.database import access as db_access
+from model.message.job import Message
+from queue_processors.queue_processor._utils_classes import _UtilsClasses
+from queue_processors.queue_processor.handling_exceptions import \
+    MissingReductionRunRecord, InvalidStateException, MissingExperimentRecord
+from queue_processors.queue_processor.settings import LOGGING
+from utils.settings import ACTIVEMQ_SETTINGS
+
+
+def validate_rb_num(rb_number):
```

Comment:
FYI: This can be refactored out into the Message class - but we'll do that in a different PR

Reply:
Opened #652